tests/manage/monitoring/test_workload_fixture.py

```diff
@@ -53,9 +53,54 @@
 @pytest.mark.libtest
 def test_workload_rbd(workload_storageutilization_50p_rbd):
     """
-    Purpose of this test is to make the workload fixture executed.
+    Purpose of this test is to make the workload fixture executed, and
```

Comment:
Are you planning to utilize the implementation here for a test case?

Reply:
Not directly. But actual test code will be based on experience gathered when writing this.